tests/test_make.py

```diff
@@ -38,6 +38,45 @@ def dataclass_param(request):
     return request.param
 
 
+def _assert_load(schema, loaded, dumped):
+    assert schema.load(dumped) == loaded
+
+
+def _assert_dump(schema, loaded, dumped):
+    assert schema.dump(loaded) == dumped
+
+
+def _assert_dump_load(schema, loaded, dumped):
+    assert schema.loads(schema.dumps(loaded)) == loaded
+
+
+def _assert_load_dump(schema, loaded, dumped):
+    assert schema.dump(schema.load(dumped)) == dumped
+
+
+def fixture_from_dict(name, id_to_value):
+    @pytest.fixture(
```

Comment:
Could this get a docstring? If you feel like it maybe some type hints `name: str, id_to_value: t.Dict[str, something]`

Reply:
Yeah, you will presumably have to bug me about type hints regularly. And documentation...

Reply:
I have no idea how far you want to go with typing so I just went, well, pretty far in on it.  Don't hesitate to guide.

Reply:
Hah! That's more than I was gonna do, but can't hurt I guess?